src/TableV8/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { ColumnDef, getCoreRowModel, useReactTable } from '@tanstack/react-table';
+import cx from 'classnames';
+import { StyledTable, StyledTbody, StyledTr, StyledTrGroup, StyledTd } from './styled';
+
+interface ReactTableProps<TData> {
+  className?: string;
+  columns: ColumnDef<TData>[];
+  data: TData[];
+  isStriped?: boolean;
+}
+
+export const TableV8 = <TData extends object>({
+  className,
+  columns,
+  data,
+  isStriped = false,
+}: ReactTableProps<TData>) => {
+  const table = useReactTable({
+    columns,
+    data,
+    getCoreRowModel: getCoreRowModel(),
+  });
+
+  return (
+    <StyledTable className={cx('rt-table', className)} role="grid">
+      <StyledTbody className="rt-tbody">
+        {table.getRowModel().rows.map((row, rowIndex) => (
+          <StyledTrGroup key={row.id} className="rt-tr-group" role="rowgroup">
+            <StyledTr
+              className={`rt-tr ${rowIndex % 2 ? '-even' : '-odd'} ${isStriped ? '-striped' : ''} `}
```

Comment:
code hygiene comment because classnames/styling gets messy quick in particular with emotion.

not sure there's a good way to do a "real" ternary with a valid if/else for css but for `isStriped` you can write a single conditional. Gets us closer to using the lib we have installed instead of sometimes doing `className={cx(...)}` and other times doing `className={`classA classb ${string interpolation}}`. Something like: 
``` css
className={cx('rt-tr', {'-striped': isStriped}, `${rowIndex % 2 ? '-even' : '-odd'}`}
```

(it will be similar in `clsx` which is nearly a drop in replacement)

Reply:
I agree with using cx over interpolation; and just in case the scenario ever comes up (as I found after struggling a bit):

cx('foo', {'--bar': true}) will produce 'foo --bar'.
if we want 'foo foo--bar', then cx('foo', {'foo--bar': true}) is needed